Dockerfile

```diff
@@ -23,8 +23,9 @@ COPY requirements_on_edge.txt .
 
 # Need to install CVToolkit separately to prevent it from re-installing Torch
```

Comment:
This comment belongs to the next RUN command

Reply:
Also not needed anymore once we merge [this PR](https://github.com/Computer-Vision-Team-Amsterdam/CVToolkit/pull/14) :)

Then we can simply add cvtoolkit to the requirements.txt again. Also for the x64 docker.